src/Morphic-Base/Form.extension.st

```diff
@@ -12,13 +12,13 @@ Form >> asMorph [
 ]
 
 { #category : #'*Morphic-Base' }
-Form class >> floodFillTolerance [
-	^ FloodFillTolerance ifNil: [FloodFillTolerance := 0.0]
+Form >> floodFillTolerance [
+	^ self class floodFillTolerance
 ]
 
 { #category : #'*Morphic-Base' }
-Form >> floodFillTolerance [
-	^ self class floodFillTolerance
+Form class >> floodFillTolerance [
+	^ FloodFillTolerance ifNil: [FloodFillTolerance := 0.0]
```

Comment:
Are these changes relevant to #ifError: refactoring? Or it comes from different commit...

Reply:
No, it's mystery to me how did this change get there.

Reply:
It's just the order that got changed. Probably just a change in the tonal format. I think its no biggy :)